examples/superdense_coding.py

```diff
@@ -0,0 +1,74 @@
+"""Superdense Coding.
+Superdense Coding is a method to transmit two classical bits of information
+from a sender to a receiver by sending only one qubit. This is accomplished
+by pre-sharing an entangled qubit.
+
+The following example sets qubit 0 to 0 and qubit 1 to 1 and are output in
+qubits 3 and 4.
```

Comment:
This whole phrase is difficult to parse :)

Reply:
Thanks very much for the feedback!  I just revised the language to hopefully make it easier to understand. I also agree with additional comment on making the input bits random and will get started on that right away.

Thanks again!